virttest/qemu_vm.py

```diff
@@ -2230,7 +2230,8 @@ def sort_key(dev):
             cmd = add_watchdog(devices,
                                params.get("watchdog_device_type", None),
                                params.get("watchdog_action", "reset"))
-            devices.insert(StrDev('watchdog', cmdline=cmd))
+            devices.insert(StrDev('watchdog', cmdline=cmd,
+                                  parent_bus={'aobject': 'pci.0'}))
```

Comment:
Not all watchdog devices are pci devices.

Reply:
You are right,  I only checked three watchdog_device_type:
1. `i6300esb`:  pci device
2. `ib700`: non pci device
3: `diag288`: for s390x only 
So, we only need to handle i6300esb, any other comments ?

Reply:
https://github.com/avocado-framework/avocado-vt/blob/a284e91d9f43b7289fffe94887d85ea794cb28ed/virttest/qemu_devices/qcontainer.py#L626-L634

Reply:
@yanan-fu sounds good to me.

Reply:
> https://github.com/avocado-framework/avocado-vt/blob/a284e91d9f43b7289fffe94887d85ea794cb28ed/virttest/qemu_devices/qcontainer.py#L626-L634

@PaulYuuu  This method does not work for watchdog, since watchdog device do not use `-device watchdog`

Reply:
@yanan-fu Yes, but it is include in the device help output
```
/usr/libexec/qemu-kvm -device ?
...
Misc devices:
name "i6300esb", bus PCI
name "ib700", bus ISA
name "isa-debug-exit", bus ISA
name "isa-debugcon", bus ISA
name "ivshmem-plain", bus PCI, desc "Inter-VM shared memory"
name "pc-testdev", bus ISA
name "pci-testdev", bus PCI, desc "PCI Test Device"
name "pvpanic", bus ISA
name "usb-redir", bus usb-bus
name "vfio-pci", bus PCI, desc "VFIO-based PCI device assignment"
name "vhost-vsock-device", bus virtio-bus
name "vhost-vsock-pci", bus PCI
name "virtio-balloon-device", bus virtio-bus
name "virtio-balloon-pci", bus PCI, alias "virtio-balloon"
name "virtio-rng-device", bus virtio-bus
name "virtio-rng-pci", bus PCI, alias "virtio-rng"
name "vmcoreinfo"
name "vmgenid"
```

Reply:
@PaulYuuu  Take `i6300esb` as an example,  It is true that QEMU support `-device i6300esb`, however when add watchdog, we use `-watchdog i6300esb`.

It is hard for me to conclude that they are equipment.